Sources/Sentry/SentryUIViewControllerSwizzling.m

```diff
@@ -222,6 +224,24 @@ - (void)swizzleRootViewControllerAndDescendant:(UIViewController *)rootViewContr
     }
 }
 
+/**
+ * We need to swizzle UIViewController 'loadView'
+ * because we can`t do it for controllers that use Nib files.
```

Comment:
`m`: Can we please add the info on why we can't do that. We can grab the info from https://github.com/getsentry/sentry-cocoa/pull/1277

> When using an UIViewController with a nib, UIKit only loads the nib if the loadView method
is not overridden. Our swizzling confused the UIKit, and therefore, it didn't load the nib
leading to an empty view and then to a crash.

Or we just add see `swizzleLoadView`, because there the code docs explain it.